lib/mpl_toolkits/mplot3d/tests/test_axes3d.py

```diff
@@ -1158,6 +1177,7 @@ def test_world():
 def test_autoscale():
     fig, ax = plt.subplots(subplot_kw={"projection": "3d"})
     assert ax.get_zscale() == 'linear'
+    ax._view_margin = 0
```

Comment:
Why is this required? Will skipping this lead to problems? It seems like this is a potential behavior change?

Reply:
Previously, the 1/48 view margin on both the upper and lower sides of an axis was applied in rendering only, so that an axis limit in the range (0, 1) would report its limits from `get_w_lims()` as (0, 1), but visually show as (-0.020833333333333332, 1.020833333333333332) = (-1/48, 49/48). This PR does change that behavior in that `get_w_lims()` reports the actual rendered limits instead of the "set" ones. I would consider the previous behavior a bug, but this might be breaking someone's workflow somewhere. Thoughts on how to handle that?